src/rtcModule/webrtc.cpp

```diff
@@ -853,6 +906,51 @@ void RtcModule::abortCallRetry(Id chatid)
     }
 }
 
+void RtcModule::refreshTurnServerIp()
+{
+    if (mIceServerProvider.empty())
+    {
+        return;
+    }
+
+    std::string url = mIceServerProvider[0]->url;
```

Comment:
The API may return a list of servers (despite currently it only returns 1). In the past, those servers used to be stored in the `mIceServerProvider` and the old call to `setIceServers(mIceServerProvider)` iterate through all those servers. 

However, now we only do it for the first server in the list and we should be considering all of them.

I can see the problem: you only plan to cache a single URL+IP, regardless the number of servers returned by API / geLB. On top off my head, I can think of multiple URLs in the cache by simply using the `TURNSERVER_SHARD + index` as id, where the index is the position in the list for each URL?. If you see complications on this suggestion, let's discuss about it in a call, @alber2510

Reply:
Yes, I think it's possible, I will modify

Reply:
Fixed